tests/python_client/common/common_type.py

```diff
@@ -226,11 +226,11 @@
 ]
 
 """ Specially defined list """
-all_index_types = ["FLAT", "IVF_FLAT", "IVF_SQ8", "IVF_PQ", "HNSW", "DISKANN", "BIN_FLAT", "BIN_IVF_FLAT",
+all_index_types = ["FLAT", "IVF_FLAT", "IVF_SQ8", "IVF_PQ", "HNSW", "SCANN", "DISKANN", "BIN_FLAT", "BIN_IVF_FLAT",
```

Comment:
Here add a new index in the array may break other test cases which call this array including GPU related cases, need to check. (e.g.: test_search_after_different_index_with_params_gpu, test_search_after_different_index_with_params...)

Reply:
I have modified all the test cases related to all_index_types.